docs/feel-unary-tests.md

```diff
@@ -6,7 +6,10 @@ title: FEEL Language Reference: Unary Tests
 
 Unary-Tests can be used only for input entries of a decision table. They are a special kind of expression with additional operators. The operators get the value of the input expression implicitly as the first argument. 
 
-The result of the expression must be either `true` or `false`.
```

Comment:
I would keep this sentence in. Otherwise you only describe under what condition the expression evaluates to true, but you don't say what the other result is.

Reply:
Ok. I'll leave it :)